clients/spark/.gitignore

```diff
@@ -1,3 +1,7 @@
 target/
 gen/
 .bsp/
+.bloop/
+.bsp/
+.metals/
+**/metals.sbt
```

Comment:
Yikes!  We _definitely_ don't want those.

Maybe move that to clients/.gitignore?

Reply:
Done